addons/point_of_sale/models/pos_order.py

```diff
@@ -820,6 +820,7 @@ def _create_misc_reversal_move(self, payment_moves):
             'date': fields.Date.context_today(self),
             'ref': _('Reversal of POS closing entry %s for order %s from session %s', self.session_move_id.name, self.name, self.session_id.name),
             'invoice_line_ids': [(0, 0, aml_value) for aml_value in move_lines],
+            'reversed_pos_order_id': self.id
```

Comment:
@vin-odoo @dylankiss You see no other possibility to find it back without the field?  

Another option might be to do a reversed_entry_id to the closing entry.  (and that could work in stable and be more consistent overall)